src/main/kotlin/net/dungeonhub/application/commands/CarryTierCommand.kt

```
@@ -240,7 +237,7 @@ class CarryTierCommand : Extension() {
                             CarryTierConnection[carryType].authenticated().getByIdentifier(arguments.carryTier)
                                 ?: throw InvalidOptionException("carry-tier", "Carry tier doesn't exist")
 
-                        if (!arguments.category && !arguments.priceChannel && !arguments.descriptiveName && !arguments.thumbnailUrl && !arguments.priceTitle) {
+                        if (!arguments.category && !arguments.priceChannel && !arguments.descriptiveName && !arguments.thumbnailUrl && !arguments.priceTitle && !arguments.ticketPanel) {
```

detekt (1.23.8)

[warning] This condition is too complex (6). Defined complexity threshold for conditions is set to '4'

(detekt.complexity.ComplexCondition)